portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,96 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// List of sentences
+var _CONTENT = [ 
```

Comment:
Only constants should have CONSTANT_CASE else it should be camel case:

https://google.github.io/styleguide/jsguide.html#naming-constant-names